batch_test.go

```diff
@@ -46,6 +46,7 @@ func init() {
 }
 
 func TestBatchWrite(t *testing.T) {
+	t.Skip()
```

Comment:
Just remember to delete this and rebase the master. Because the performance issues had already been fixed.